internal/cli/cli.go

```diff
@@ -269,3 +269,17 @@ func getHostNameAndPort(hostInfo string) (string, int, error) {
 
 	return host[0], port, nil
 }
+
+// indexKeys returns a map of properties starting from a []string which contains properties in the format field:type
```

Comment:
this ocmment doesn't make much sense, a more helpful message would be
```suggestion
// indexKeys  takes a slice of values formatted as key:vale and returns an array of [key]:value
```